e2e/cypress/tests/integration/settings/display/message_display_mode_colorize_username_spec.js

```diff
@@ -0,0 +1,140 @@
+        cy.uiSave();
+        cy.uiClose();
+    });
+
+    it('MM-T4984_2 Message Display: colorize usernames option should exist in Compact mode and function as expected', () => {
+        // # Select 'Compact' option
+        cy.findByRole('heading', {name: 'Message Display'}).click();
+        cy.findByRole('radio', {name: 'Compact: Fit as many messages on the screen as we can.'}).click();
+
+        // * Verify Colorize usernames option exists and checked by default
+        cy.findByRole('checkbox', {name: 'Colorize usernames: Use colors to distinguish users in compact mode'}).should('exist');
+        cy.findByRole('checkbox', {name: 'Colorize usernames: Use colors to distinguish users in compact mode'}).should('be.checked');
+
+        // # Save and close the modal
+        cy.uiSave();
+        cy.uiClose();
+
+        // # Save the color of the buttons
+        cy.findByText(firstUser.username).then((elements) => {
+            colors[firstUser.username] = elements[0].attributes.style.value;
+        });
+        cy.findByText(otherUser.username).then((elements) => {
+            colors[otherUser.username] = elements[0].attributes.style.value;
+        }).then(() => {
+            // * Verify that colors are different
+            expect(colors[firstUser.username]).to.not.equal(colors[otherUser.username]);
+        });
+
+        cy.reload();
+
+        // * Verify that after reload colors are the same
+        cy.findByText(firstUser.username).then((elements) => {
+            cy.wrap(elements[0]).should('have.attr', 'style', colors[firstUser.username]);
+        });
+        cy.findByText(otherUser.username).then((elements) => {
+            cy.wrap(elements[0]).should('have.attr', 'style', colors[otherUser.username]);
+        });
+    });
+
+    it('MM-T4984_3 Message Display: disabling colorize should revert to colors to normal color', () => {
+        // # Select 'Compact' option
+        cy.findByRole('heading', {name: 'Message Display'}).click();
+        cy.findByRole('radio', {name: 'Compact: Fit as many messages on the screen as we can.'}).click();
+
+        // # Verify Colorize usernames option exists and make it unchecked
+        cy.findByRole('checkbox', {name: 'Colorize usernames: Use colors to distinguish users in compact mode'}).should('exist');
+        cy.findByRole('checkbox', {name: 'Colorize usernames: Use colors to distinguish users in compact mode'}).uncheck().should('not.be.checked');
+
+        // # Save and close the modal
+        cy.uiSave();
+        cy.uiClose();
+
+        // * Verify that colors are reverted to normal
+        cy.findByText(firstUser.username).then((elements) => {
+            cy.wrap(elements[0]).should('have.css', 'color', 'rgb(63, 67, 80)');
+        });
+        cy.findByText(otherUser.username).then((elements) => {
+            cy.wrap(elements[0]).should('have.css', 'color', 'rgb(63, 67, 80)');
+        });
+    });
+});
+
+function goToMessageDisplaySetting() {
```

Comment:
I think this function already exists

`cy.uiChangeMessageDisplaySetting()`

e2e/cypress/tests/support/ui/account_settings_modal.js

Reply:
Thanks for the recommendation. 
`cy.uiChangeMessageDisplaySetting()` is setting as `Standard` or `Compact`. `goToMessageDisplaySetting` function just goes to the screen and stays there. 

Anyhow, I used `cy.uiChangeMessageDisplaySetting()` to simply part of my code.